src/Account.sol

```diff
@@ -209,45 +209,48 @@ contract Account is IAccount, Auth, OpsReady {
             if (!isOwner()) revert Unauthorized();
 
             if (_command == Command.ACCOUNT_MODIFY_MARGIN) {
+                // Command.ACCOUNT_MODIFY_MARGIN
                 int256 amount;
                 assembly {
                     amount := calldataload(_inputs.offset)
                 }
                 _modifyAccountMargin({_amount: amount});
             } else {
+                // Command.ACCOUNT_WITHDRAW_ETH
                 uint256 amount;
                 assembly {
                     amount := calldataload(_inputs.offset)
                 }
                 _withdrawEth({_amount: amount});
             }
         } else {
-            /// @dev both owner and delegates can execute the following commands
+            /// @dev only owner and delegate(s) can execute the following commands
             if (!isAuth()) revert Unauthorized();
 
             if (commandIndex < 4) {
-                address market;
-
                 if (_command == Command.PERPS_V2_MODIFY_MARGIN) {
+                    // Command.PERPS_V2_MODIFY_MARGIN
+                    address market;
                     int256 amount;
                     assembly {
                         market := calldataload(_inputs.offset)
                         amount := calldataload(add(_inputs.offset, 0x20))
                     }
                     _perpsV2ModifyMargin({_market: market, _amount: amount});
                 } else {
+                    // Command.PERPS_V2_WITHDRAW_ALL_MARGIN
+                    address market;
                     assembly {
                         market := calldataload(_inputs.offset)
                     }
                     _perpsV2WithdrawAllMargin({_market: market});
                 }
-            } else if (commandIndex < 10) {
-                address market;
-                int256 sizeDelta;
-                uint256 desiredFillPrice;
-                bytes32 marketKey;
-
+            } else if (commandIndex < 6) {
                 if (_command == Command.PERPS_V2_SUBMIT_ATOMIC_ORDER) {
+                    // Command.PERPS_V2_SUBMIT_ATOMIC_ORDER
+                    address market;
+                    int256 sizeDelta;
+                    uint256 desiredFillPrice;
```

Comment:
Reason this was moved?

Reply:
I added it originally to reduce bytecode size but realized when I was refactoring that it makes things a bit more complicated when trying to add/remove/rearrange commands. In the future, when adding commands or altering the checks for fee optimizations, it will be easier to do so if each command's corresponding block is isolated.
Furthermore, Uniswap's Universal Router follows this principle despite many shared inputs across command types. I imagine their reason for doing so is likely the same as mine, as they are actively modifying supported commands.

Reply:
Not passionate either way - but I agree that readable, refactorable and easily extensible code is more important than small reductions in deployed bytecode size.